avm/utilities/pipelines/e2eValidation/regionSelector/Get-AzAvailableResourceLocation.ps1

```diff
@@ -65,19 +65,27 @@ function Get-AzAvailableResourceLocation {
   $ResourceRegionList = (Get-AzResourceProvider | Where-Object { $_.ProviderNamespace -eq $formattedResourceProvider }).ResourceTypes | Where-Object { $_.ResourceTypeName -eq $formattedServiceName } | Select-Object -ExpandProperty Locations
   Write-Verbose "Region list: $($resourceRegionList | ConvertTo-Json)"
 
-  $locations = Get-AzLocation | Where-Object {
-    $_.DisplayName -in $ResourceRegionList -and
-    $_.Location -notin $ExcludedRegions -and
-    $_.PairedRegion -ne "{}" -and
-    $_.RegionCategory -eq "Recommended"
-  } |  Select-Object -ExpandProperty Location
-  Write-Verbose "Available Locations: $($locations | ConvertTo-Json)"
+  if ($resourceRegionList -eq "global") {
+    Write-Verbose "Resource is global, default region [West Europe] will be used for resource group creation"
+    $location = "westeurope" # Using West Europe for resource group location. GLobabl resources should have hardocded location in `main.bicep`
```

Comment:
Instead of hardcoding this value we can get this from deploymentMetadataLocation (avm template workflow)

Reply:
Completed